paasta_tools/utils.py

```diff
@@ -1340,3 +1340,16 @@ def format_table(rows, min_spacing=2):
             expanded_rows.append(expanded_row)
 
     return [(' ' * min_spacing).join(r) for r in expanded_rows]
+
+
+def deep_merge_dictionaries(source, destination):
```

Comment:
As the comment mentions [here](https://stackoverflow.com/questions/20656135/python-deep-merge-dictionary-data) what happens if an element in A is a dict but in B is not?